cypress/e2e/with_api/systems/functions.ts

```diff
@@ -206,3 +206,15 @@ export const editSystems = () => {
     type: 'Operational',
   });
 };
+
+export const navigateToItemTableForSparesDefinition = () => {
```

Comment:
```suggestion
export const navigateToItemsTableViaSpares = () => {
```
I assume this is navigating via the number of spares to the items table, rather than for testing the spares definition?